docs/blueprints/log-in-with-storj.md

```diff
@@ -0,0 +1,205 @@
+# Log In with Storj
+
+## Background
+
+As we continue to grow, we need to enable engineers to build distributed web applications on top of the Storj platform.
+As a third party application developer, I rarely want to deal with user management. I often rely on third party services
+(like Google, GitHub, etc) to handle authentication for me. Beyond authentication, this type of integration often allows
+my application to access information from the authenticating service on the user's behalf. For example, if I were to log
+in with GitHub, the requesting application can be granted permissions to read private repositories, packages, and more.
+
+## Abstract
+
+In this document, I describe how we can turn the Satellites into an OpenID Provider. [OpenID Connect][] (or OIDC)
+provides a slim identity layer (authentication) on top of [OAuth2][] (authorization). This allows new applications to be
+developed against the core of the Storj platform, using well know, well-supported, best practices. By integrating with
+OIDC, we ultimately put the control over ones identity in hands of our end users. Instead of passing around and storing
+access grants, applications and satellites can negotiate time-bound access and refresh tokens on behalf of authenticated
+end users.
+
+To help solidify the concepts in this review, I've included diagrams to illustrate interactions between Storj, StorJIF -
```

Comment:
`StorJIF` - I think it's useful to use a service name that doesn't use "Storj" to clearly indicate that this is an external project.

Reply:
Swapped this for a non stroj name